docs/content/stable/yugabyte-platform/manage-deployments/retry-failed-task.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Use YugabyteDB Anywhere to retry failed tasks
+headerTitle: Retry a universe task
+linkTitle: Retry a universe task
+description: Use YugabyteDB Anywhere to retry failed tasks.
+menu:
+  stable_yugabyte-platform:
+    identifier: retry-task
+    parent: manage-deployments
+    weight: 75
+type: docs
+---
+
```

Comment:
What about Tasks tab? Tasks are listed there; can you retry from there as well?

Show both methods, and introduce the Tasks tab generally as a centralized spot for checking this stuff